universum/modules/vcs/perforce_vcs.py

```diff
@@ -105,7 +113,7 @@ def disconnect(self):
                 return
             text = ""
             for line in w:
-                text += "\n" + warnings.formatwarning(line.message, line.category, line.filename, line.lineno)
+                text += "\n" + warnings.formatwarning(str(line.message), line.category, line.filename, line.lineno)
```

Comment:
mypy was complaining that line.message is a Warning - not a str

Reply:
Can you check it?

Reply:
reveal_type(line.message)

> universum/modules/vcs/perforce_vcs.py:116: note: Revealed type is 'builtins.Warning'

Reply:
How it was working before?
Is it some sort of automatic conversion to str inside `formatwarning`?

Reply:
I think so